requirements/local.txt

```diff
@@ -274,5 +274,3 @@
 # The following packages are considered to be unsafe in a requirements file:
 # setuptools
 
-elasticsearch-dbapi
```

Comment:
Just checking. Safe to delete?

Reply:
Yes, this was a hack that was put in place early on for dev purposes. This is not the right location to include these, so they were moved when I split the build in two.

Reply:
Yup needs to not be there. I was not there in cccs-1.2